src/main/java/entities/Comment.java

```diff
@@ -1,36 +1,26 @@
 package entities;
 
-
-import java.util.ArrayList;
 import java.util.Date;
-import java.util.List;
 
 /**
  * Comment extends abstract class Postable and inherits all of its
  * instance attributes (user, body, replies) and addComment method.
  *
  */
 public class Comment extends Postable{
-    // number of Comments instantiated
-    public static int numComments = 0;
-    // id is the unique identifier of Comment, equal to number of Comments at time of instantiation
-    private final int id;
 
     /**
      * Initializes an instance of Comment.
      *
      * @param commenter The user that posted this comment.
      * @param body The text content of the comment.
-     * @param numCommentsCreated The number of comments created in the program so far.
+     * @param id The unique id for this comment
      */
-    public Comment(User commenter, String body, int numCommentsCreated){
+    public Comment(User commenter, String body, int id){
```

Comment:
we need a second constructor for Comment that takes parameters:
- User commenter
- String body
- int id
- String creationDate

Reply:
@izumidayo I think that the creationDate should be a LocalDate, no? It would allow for date methods if needed.

Reply:
I think a second construct would only be necessary if we need to store/edit a comment and put it back into the database which I don't think will happen(comments never get edited since it does not have replies now).

creationDate is a string because when we put posts back into the database, the default initialization of date changes it to the time it was put back in, instead of the actual creation date which we want, since creationDate is from postable, comment is also forced to keep it as a string. If we need to use date methods, I think converting a date formated as a string back into a date would be easy.